assets/app/views/_triggers.html

```diff
@@ -1,11 +1,25 @@
 <div class="builds" ng-repeat="trigger in triggers">
   <div ng-repeat="build in trigger.builds" ng-if="(build.status != 'Complete' && build.status != 'Failed') || (build.metadata.creationTimestamp | ageLessThan : 5 : 'minutes')" class="build animate-repeat osc-object" kind="Build" resource="build">
+    <!-- Icon -->
     <span ng-switch="build.status" class="hide-ng-leave">
       <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true" style="margin-right: 5px;"></span>
       <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
+      <span ng-switch-when="Error" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
+      <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
       <span ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true" style="margin-right: 5px;"></span>
     </span>
-    <span>A build of {{build.metadata.labels.buildconfig}} is {{build.status | lowercase}}.</span>
-    <span ng-if="build.status != 'Complete' && build.status != 'Failed' && trigger.imageChangeParams.automatic">  A new deployment will be created automatically once the build completes.</span>
+    <!-- Message -->
+    <span ng-switch="build.status" class="hide-ng-leave">
+      <span ng-switch-when="Complete">A build of {{build.metadata.labels.buildconfig}} completed.</span>
```

Comment:
factor out `A build of {{build.metadata.labels.buildconfig}}` so we don't have to repeat it. Error can say "... encountered an error."